lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -869,6 +869,81 @@ namespace Js
     }
 
     template <typename T>
+    BOOL DictionaryTypeHandlerBase<T>::DeleteProperty(DynamicObject *instance, JavascriptString *propertyNameString, PropertyOperationFlags propertyOperationFlags)
```

Comment:
I am also worried about have "counterpart" code you added in all the files. In future, people might make changes in one of the method and forget making similar change in its counterpart. Instead is there a way to extract out common part in a method that both properytNameString and propertyId would call?

---
In reply to: [78818261](https://github.com/Microsoft/ChakraCore/pull/1542#discussion_r78818261) [](ancestors = 78818261,78817010,78791124)